privaterelay/views.py

```diff
@@ -64,10 +64,16 @@ def profile(request):
     )
     fxa_account = request.user.socialaccount_set.filter(provider='fxa').first()
     avatar = fxa_account.extra_data['avatar'] if fxa_account else None
+    context = {'relay_addresses': relay_addresses, 'avatar': avatar}
 
-    return render(request, 'profile.html', {
-        'relay_addresses': relay_addresses, 'avatar': avatar
-    })
+    profile = request.user.profile_set.first()
+    bounce_paused, bounce_type = profile.check_bounce_pause()
+    if bounce_paused:
+        context.update({'last_bounce_type': bounce_type})
+        context.update({'last_bounce_date': profile.last_bounce_date})
+        context.update({'next_email_try': profile.next_email_try})
+
+    return render(request, 'profile.html', context)
```

Comment:
if you ever let me at this code base, we will be using class-based views :)

Reply:
you're welcome to the code-base! we can have the implicit code flow argument anytime.